### PR TITLE
[JA] Add `silence`

### PR DESCRIPTION
**Pull request by yuzupon1133**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

japanese reviewer where

### CODE REVIEW

**Review by peppy — CHANGES_REQUESTED** [review]
Just some minor additions and nuance changes

File: wiki/Silence/ja.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+## サイレンスの異議申し立て
+
+もしあなたが不当にサイレンスを受けたと思うのであれば、異議申し立てを[アカウントサポートチーム](/wiki/People/The_Team/Account_support_team#accounts@ppy.sh)の[accounts@ppy.sh](mailto:accounts@ppy.sh)まで連絡してください。その際、あなたのosu!アカウントにリンクされたメールアドレスから、ユーザー名とサイレンスにを受けた大体の時間を明記してください。
+
+## サイレンスの一般的な理由
+
+注: プライベートメッセージで「不適切な言動」を行うことは、その不適切な言動が不本意または同意のない受信者に送信しない限り、ルール違反ではありません。報告された場合は処分が下されます。
+
+- パブリックチャンネルやフォーラム（ビートマップ、コメントなど）で不適切な言動（スパム、無意味なチャット、人種差別などを含むがこれに限らない）を行うこと
+- 他の手段で不適切なコンテンツをアップロードまたは投稿すること(例：プロフィールアイコン、ビートマップ、プロフィールの詳細、カバー画像など）
```
[the remark this answers is not in the document]
Comment (yuzupon1133) [reply wiki/Silence/ja.md:38]: Is this the banner mean?
image

[the remark this answers is not in the document]
Comment (peppy) [reply wiki/Silence/ja.md:38]: Yeah. I'm fine with your terminology, can leave it as you had it. I think カバー was used elsewhere but I forget where I checked..

[the remark this answers is not in the document]
Comment (yuzupon1133) [reply wiki/Silence/ja.md:38]: In Japanese, the "banner" is "カバー画像" on website. I think.
image

**Review by peppy — APPROVED** [review]

**Pull request merged**